experiments/bid_vs_cot.py

```
@@ -0,0 +1,342 @@
+    print("\n🔎 Evaluating BASELINE strategy...")
+    baseline = evaluate_strategy("baseline", baseline_prompt)
+    
+    print("\n🔎 Evaluating CONSTITUTION strategy...")
+    constitution = evaluate_strategy("constitution", constitution_prompt)
+    
+    # ---------- REPORT ----------
+    print("\n" + "="*60)
+    print("FINAL REPORT")
+    print("="*60)
+    
+    print(f"\nBASELINE:")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)